### PR TITLE
make post

### PR DESCRIPTION
**Pull request by nam040494**

### CODE REVIEW

**Comment by nam040494** [comment]
ready

File: app/views/posts/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <div id="error_explanation">
+      <h2>
+        <%= "#{pluralize(@post.errors.count, "error")}" + t(".text") %>
+      </h2>
+      <ul>
+        <% @post.errors.full_messages.each do |msg| %>
+          <li>
+            <%= msg %>
+          </li>
+          <% end %>
```
Comment (huongnguyenmta) [inline app/views/posts/_form.html.erb]: indent

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <% @posts.each do |post| %>
+      <span class="user">
+        <%= image_tag post.user.avatar.url :thumb %>
+        <%= link_to post.user.name, post.user %>
+      </span>
+      <span class="content">
+        <h2><%= post.title %></h2>
+        <h6><%= post.created_at.strftime("%b %d, %Y") %></h6>
+        <%= raw(post.content).truncate(358) %>
+        <%= link_to "READ MORE", post_path(post) %>
```
Comment (huongnguyenmta) [inline app/views/posts/index.html.erb]: I18n

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="posts">
+    <% @posts.each do |post| %>
+      <span class="user">
+        <%= image_tag post.user.avatar.url :thumb %>
+        <%= link_to post.user.name, post.user %>
+      </span>
+      <span class="content">
+        <h2><%= post.title %></h2>
+        <h6><%= post.created_at.strftime("%b %d, %Y") %></h6>
+        <%= raw(post.content).truncate(358) %>
```
Comment (huongnguyenmta) [inline app/views/posts/index.html.erb]: move `385` into setting

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="posts">
+    <% @posts.each do |post| %>
+      <span class="user">
+        <%= image_tag post.user.avatar.url :thumb %>
+        <%= link_to post.user.name, post.user %>
+      </span>
+      <span class="content">
+        <h2><%= post.title %></h2>
+        <h6><%= post.created_at.strftime("%b %d, %Y") %></h6>
```
Comment (huongnguyenmta) [inline app/views/posts/index.html.erb]: move `%b %d, %Y` into setting

**Comment by huongnguyenmta** [comment]
commented

**Comment by nam040494** [comment]
ready

**Comment by huongnguyenmta** [comment]
merged
Mớt cưng nha... (dance2)

**Pull request merged**